src/main/java/sirius/kernel/xml/Outcall.java

```diff
@@ -52,8 +54,9 @@
  */
 public class Outcall {
 
-    private static final int DEFAULT_CONNECT_TIMEOUT = (int) TimeUnit.MILLISECONDS.convert(5, TimeUnit.MINUTES);
+    private static final int DEFAULT_CONNECT_TIMEOUT = (int) TimeUnit.MILLISECONDS.convert(30, TimeUnit.SECONDS);
```

Comment:
think this could be a problem with OX and session migration

Reply:
this is a default and can be overwritten any time - 5m timeout is an absolute server killer